www/scripts/main.js

```diff
@@ -0,0 +1,187 @@
+﻿var userNameField = $("#name");
```

Comment:
I think there's a small chance this code could be executed before the dom is loaded which will cause some annoying bugs. I think it's worth moving all the ```$(...)``` inside a ```$(function(){...})``` so we can be assured that the dom is ready.